### DISCUSSION TITLE
[V4] Get table name from model

### DISCUSSION THREAD

**Discussion by Agilulfo1820** [adonisjs/core#2792] [Help] [answered]
Hello, let's say I have the `User` model, is there a way I can get the table name (so `users`)?

Something like `model.table()`.

**Comment by RomainLanz** [comment, marked as answer]
Hey @Agilulfo1820! 👋 

You should be able to call [`User.table`](https://github.com/adonisjs/lucid/blob/legacy/src/Lucid/Model/index.js#L184) to get the default tables name.

**Reply by Agilulfo1820** [reply]
thanks, it worked.
